apps/docs/src/routes/docs.tsx

```diff
@@ -3,6 +3,7 @@ import { Outlet, createFileRoute } from "@tanstack/react-router"
 import { DocsSidebarNav } from "~/components/sidebar-nav"
 import type { SidebarNavItem } from "~/config/docs"
 import { components, general_pages } from "#site/content"
+import { sortUserPlugins } from "vite"
```

Comment:
This is not used right?

Reply:
oops, unused wrong import..
is deleted ✅